packages/snfoundry/scripts-ts/helpers/deploy-wrapper.ts

```diff
@@ -6,22 +6,32 @@ interface CommandLineOptions {
   _: string[]; // Non-hyphenated arguments are usually under the `_` key
   $0: string; // The script name or path is under the `$0` key
   network?: string; // The --network option
+  reset?: boolean;
+  fee?: string;
```

Comment:
Now we can deploy our SC using STRK for the fee, we will have to indicate the command `--fee strk` for example, `yarn deploy --fee strk`